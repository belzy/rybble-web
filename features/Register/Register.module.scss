.register {
  color: gray;
  border: 1px solid red;
}