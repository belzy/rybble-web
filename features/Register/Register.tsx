import React from 'react';
import { AuthPage, Link } from '../../components';

const Register = () => {

  return (
    <AuthPage></AuthPage>
  );
};
export default Register;