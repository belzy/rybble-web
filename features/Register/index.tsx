import React from 'react';
import './Register.module.scss';

const x = 0;

x;


const Register = () => {

  return ( 
    <div className="register">
      <h1>Sign In</h1>
    </div> 
  );
};
export default Register;