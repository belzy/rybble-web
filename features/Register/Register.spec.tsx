import { render, screen } from '@testing-library/react';
import Register from '.';
import '@testing-library/jest-dom';

describe('features/Register', () => {

  it('should render Register component unchanged', () => {

    const { container } = render(<Register />);

    expect(container).toMatchSnapshot();

  });

  it('should render a heading', () => {

    render(<Register />);

    const heading = screen.getByRole('heading', {
      name: /Sign In/i,
    });

    expect(heading).toBeInTheDocument();

  });

});