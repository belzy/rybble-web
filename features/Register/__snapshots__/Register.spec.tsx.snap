// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`features/Register should render Register component unchanged 1`] = `
<div>
  <div
    class="register"
  >
    <h1>
      Sign In
    </h1>
  </div>
</div>
`;
